css-nesting-1/Overview.bs

```diff
@@ -466,11 +466,12 @@ Nesting Conditional Rules {#conditionals}
 -----------------------------------------------
 
 	A style rule can have any number of <a>conditional rules</a> inside of it, of any type,
-	intermixed with any number of declarations, in any order.
+	with rules required to come after declarations (see <a href="#mixing">Mixing Nesting Rules and Declarations</a>).
 
-	The presence of a nested conditional engages the same logic as if ''@nest'' was present.
-	The nested conditional rules must contain a <a>nesting selector</a> and follow the same
-	rules as outlined in <a href="#direct">direct nesting</a>.
+	If there are only declarations in the nested conditional, the nesting context is adopted.
```

Comment:
```suggestion
	All valid declarations in the nested conditional adopt the nesting context.
```

Reply:
is that clearer @LeaVerou? hopefully it's less restrictive sounding, that nested declarations adopt context and discovered nesting selectors follow the nesting logic path.

Reply:
Yes, though I wonder if it's clearer to word this in terms of syntactic sugar, i.e. that if there are any declarations inside the conditional rule, there is an implicit `& { ... }` around all of them.

Reply:
i could add that to the examples where we're saying x is equivalent to y? with nested conditional examples showing the explicit in-between state which includes the `&`? or you think it's better in the text inline here?